api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -103,8 +104,15 @@ public void syncBillingProjectStatus() {
           default:
             break;
         }
+      } catch (NotFoundException e) {
+        log.log(Level.WARNING, "Get BillingProjectStatus call failed for " + entry.getCreationTime() + ". Project not found.");
+
+        if ((getCurrentTimestamp().getTime() - entry.getCreationTime().getTime()) > minutesToMs(CREATING_TIMEOUT_MINUTES)) {
```

Comment:
Is this necessary? I think `cleanBillingBuffer` already covers this without modification.